content/vendor/guides/configuring-application-snapshots.md

```diff
@@ -0,0 +1,345 @@
+            - sh
+            - -c
+            - exec pg_isready --host $POD_IP
+          initialDelaySeconds: 5
+          timeoutSeconds: 3
+          periodSeconds: 5
+        resources:
+          requests:
+            cpu: 100m
+            memory: 256Mi
+
+        volumeMounts:
+        - name: postgresql-vct
+          mountPath: /var/lib/postgresql/data/pgdata
+          subPath: postgresql-db
+        - name: postgresql-backup
+          mountPath: /backup
+          subPath: postgresql-db
+  volumeClaimTemplates:
+  - metadata:
+      name: postgresql-vct
+    spec:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 2Gi
+  - metadata:
+      name: postgresql-backup
+    spec:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 2Gi
+
+```
+{{< notes title="Note" >}}
+Note that in the command in the label annotation we are backing up the appdirectdb database, which does not exist when the application is first deployed. 
+For testing purposes, you could change it to back up the postgres database.
+{{< /notes >}}
+### Configuring the Cluster
+
+KOTS uses the [Velero](https://velero.io/) open source project to provide back up capabilities and will need to be added to the cluster. 
+The process for adding Velero will depend if the cluster is being managed by KOTS or not.
+
+#### Clusters Managed by KOTS
+
+For those applications running on a cluster [managed by KOTS](https://kots.io/vendor/packaging/embedded-kubernetes/), add the Velero add-on to the Kubernetes KURL installer. 
+While there are various configuration options as outlined [here](https://kurl.sh/docs/add-ons/velero), you should only need to add the following to your KURL Kubernetes installer.
+
+```shell
+  velero:
+    version: latest
+```
+
+Make sure to promote the installer to the same channel as the application.
+
+#### Other Clusters
+
+Since KOTS does not manage the cluster in this instance, it has to be done by the end user and covered in [Part II](#check-velero-installation).
+
+## Part II - Application Configuration
+
+Once the configurations covered in Part I have been pushed to a release and promoted, anyone that installs or updates the application to this release will now have the ability to configure Snapshots. 
+The user will know that Snapshots are now available to configure when the **Snaphot Settings** tab becomes avialabe as illustrated below.
+
+![initialwindow](/images/guides/kots/snap_guide_initial_window.png)
+
+To configure snapshpots in the Admin Console, click on the **Snapshot Settings** tab. 
+Here, we'll configure settings that apply to this instance of KOTS which allows you to specify a Storage Option and check the Velero install.
+
+![storage_opts](/images/guides/kots/snap_guide_storage_conf.png)
+
+#### Choose Storage
+
+There are three options currently available to store the snapshots:
+
+- Amazon S3 Bucket
+- S3 Compatible Bucket
+- Local Storage
```

Comment:
This is another case where, since this is a tutorial, why don't we give the user some guidance, or at least just tell them what to pick.